### PR TITLE
Prevent compiler-arguments tests from polluting the repo

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: tests/ui/ferrocene/compiler-arguments/o/o_no-delim.rs
Comment (Veykril) [inline tests/ui/ferrocene/compiler-arguments/o/o_no-delim.rs:1]: Removed as this is a duplicate of `o_o.rs`, exact same files

Comment (tshepang) [reply tests/ui/ferrocene/compiler-arguments/o/o_no-delim.rs:1]: I suppose you referring to `o.rs`, and even then, it's not quiet the same... one has space in the cli (which I suppose it what is referred to as delim)

Comment (Veykril) [reply tests/ui/ferrocene/compiler-arguments/o/o_no-delim.rs:1]: I did mean `o.rs` right, but neither have a space in them, both just pass `-ofoo`

Comment (tshepang) [reply tests/ui/ferrocene/compiler-arguments/o/o_no-delim.rs:1]: oh, indeed...

**Review by tshepang — APPROVED** [review]
bors merge

**Pull request merged**